adapters/src/main/java/com/squareup/moshi/adapters/PolymorphicJsonAdapterFactory.java

```diff
@@ -205,19 +197,15 @@
   }
 
   private JsonAdapter<Object> buildFallbackJsonAdapter(final T defaultValue) {
-    return new DefaultJsonAdapter<Object>() {
-      @Nullable
-      @Override
-      public Object fromJson(JsonReader reader) throws IOException {
+    return new JsonAdapter<Object>() {
+      @Override public @Nullable Object fromJson(JsonReader reader) throws IOException {
```

Comment:
```suggestion
      @Nullable @Override public Object fromJson(JsonReader reader) throws IOException {
```

Reply:
`@Nullable` is a type annotation and annotates the return type`Object` and not the method. Like `?` in Kotlin!

Reply:
I know, it was a style comment. Looking through the rest of the codebase, seems moshi uses a mix of both annotating before and after the modifiers 😅